pkg/authorization/authorizer/authorizer.go

```diff
@@ -52,7 +52,17 @@ func (a *openshiftAuthorizer) Authorize(ctx kapi.Context, passedAttributes Autho
 		return false, "", kerrors.NewAggregate(errs)
 	}
 
-	return false, "denied by default", nil
+	username := "system:MISSING"
```

Comment:
don't make this look like a real username... make it "Unknown user" or something